charts/featureform/values.yaml

```diff
@@ -57,6 +57,16 @@ dashboardUrl: ""
 # If true, restarts pods on update even if no changes have been made
 restartOnUpdate: false
 
+argo:
+  # Enables path forwarding of /argocd/ to the Argo web server. Note this still requires installation of Argo,
+  # done outside of this chart itself as it is a global service across the cluster.
+  enabled: false
```

Comment:
couple of q's: 
- is the argo update an all-or-nothing prop? is it easy to revert back and forth between argo being enabled or not (given the flag in the values file)

- we have a few jobs resources that we use in the app, do we use them normally or do they need to be "argo-fied" too.

Reply:
Gotcha, ok I'll pass along the jobs to you, although it's possible we might change how the data_loader works in the future.